python/core/fundamental-data-structures/py-tuple-assignment.md

```diff
@@ -0,0 +1,114 @@
+---
+author: kapnobatai136
+
+tags:
+  - introduction
+
+type: normal
+
+category: must-know
+
+aspects:
+  - introduction
+
+links:
+  - '[Tuples and Sequences](https://docs.python.org/3.5/tutorial/datastructures.html#tuples-and-sequences){website}'
+
+---
+
+# Tuple Assignment
+
+---
+## Content
+
+The **tuple assignment** feature allows packing and unpacking such that a tuple left of the assignment will be assigned all values from the tuple right of the assignment:
+```python
+flash = ("Berry", "Allen" , 27)
+(fname, lname, age) = flash
+print(fname)
+# 'Berry'
+print(age)
+# 27
+```
+
+In case of tuples with multiple fields, use the `*` unpacking syntax:
+```python
+a, *b, c = (1, 2, 3, 4, 5)
```

Comment:
above you introduced the `(a, b, c)` syntax and here we drop the paranthesis. we should probably explain why & what's the difference

Reply:
there's no difference. when defining tuples you can omit the parentheses. I've included this in the insights